azure-pipelines.yml

```diff
@@ -332,47 +309,12 @@ stages:
 
       displayName: Deploy Elasticsearch
 
-  - job: e2e_image
-    displayName: Prepare E2E Image
-    steps:
-
-    - task: AzureCLI@1
-      displayName: Login to ACR
-      inputs:
-        azureSubscription: $(ACR_PUSH_SERVICE_CONNECTION)
-        scriptLocation: inlineScript
-        inlineScript: |
-          set -eux  # fail on error
-
-          az configure --defaults acr="$ACR_NAME"
-          az acr login
-
-    - bash: |
-          # Build end-to-end test image
-          docker build --target end2endtest \
-            --build-arg VersionPrefix="$SEMANTIC_VERSION" \
-            -t "$ACR_NAME.azurecr.io/k2bridge-test:$SEMANTIC_VERSION" \
-            .
-      displayName: Build E2E-Test Image
-
-    - task: AzureCLI@1
-      displayName: Push Image
-      inputs:
-        azureSubscription: $(ACR_PUSH_SERVICE_CONNECTION)
-        scriptLocation: inlineScript
-        inlineScript: |
-          docker push "$ACR_NAME.azurecr.io/k2bridge-test:$SEMANTIC_VERSION"
-
```

Comment:
See my comment above in the build stage.
How about keeping this here to save time?